e2e/test/scenarios/permissions/permissions-reproductions.cy.spec.js

```diff
@@ -164,45 +164,6 @@ describeEE("postgres > user > query", { tags: "@external" }, () => {
   });
 });
 
-describeEE("issue 17763", () => {
```

Comment:
now captured in a more generic test "should preserve parent value for children when selecting granular for permissions available to child entities"